<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMeetingDialog</class>
 <widget class="QDialog" name="CreateMeetingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreateMeetingDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>611</width>
     <height>281</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>371</width>
     <height>271</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="title">
    <string>Meeting Info</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Start date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>End date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startDateEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="startTimeEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="endTimeEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDateEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="comfirmButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bell MT&quot;;</string>
    </property>
    <property name="text">
     <string>Comfirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bell MT&quot;;</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Participator </string>
    </property>
   </widget>
   <widget class="QTableView" name="userTableView">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>191</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Aparajita&quot;;</string>
    </property>
    <property name="text">
     <string>perss Ctrl to choose more than one</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
